parachain/runtime/src/lib.rs

```diff
@@ -382,7 +382,7 @@ impl parachain_tokens::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 500;
+    pub const ExistentialDeposit: u128 = 1;
```

Comment:
Not `0`?

Reply:
In all of the mocks it was set to `1`, which I guess makes sense - if a balance becomes less than one, it's ok if the account gets removed